scoresheets/HelpMeCarry.tex

```diff
@@ -1,52 +1,20 @@
 The maximum time for this test is 5 minutes.
 
-{\footnotesize
 \begin{scorelist}
-	\scoreheading{Following Phase} % 30 pts
-	\scoreitem{10}{Follow operator outside the arena}
-	\scoreitem{15}{Follow operator to the car}
-	\scoreitem{ 5}{Understand the destination}
-
-\ifNotSSPL{
-	% These are mutually exclusive, max score is thus 20
-	\scoreheading{Bag pick-up (OPL \& DSPL only)}
-	\scoreitem{+0}{Bag hanged. Gripper closes on timeout}
-	\scoreitem{ 2}{Bag hanged. Gripper closes on hang}
-	\scoreitem{ 5}{Pick up the bag from the floor}
-	\scoreitem{10}{Scripted handover (hand/bag detection only)}
-	\scoreitem{20}{Natural handover (active grasping + object release detection)}
-
-	\scoreheading{DSPL \& OPL Tasks} % 80 pts
-	\scoreitem{10}{Re-enter the arena}
-	\scoreitem{ 5}{Deliver the bag at the specified location}
-	\scoreitem{10}{Find the person at the specified location}
-	\scoreitem{30}{Open door without help}
-	\scoreitem{10}{Guide operator outside the arena}
-	\scoreitem{15}{Guide operator to the car}
-}
-
-\ifSSPL{
-	\scoreheading{SSPL only Tasks} % 100 pts
-	\scoreitem{10}{Tell the time to the stranger}
-	\scoreitem{30}{Re-enter the arena}
-	\scoreitem{20}{Find the person at the specified room}
-	\scoreitem{10}{Guide operator outside the arena}
-	\scoreitem{30}{Guide operator to the car}
-}
-
-	\scoreheading{Obstacle avoidance} % 70 pts
-	\scoreitem{20}{Avoiding small (box-sized) object}
-	\scoreitem{20}{Avoiding 3D (hard-to-see) object}
-\ifSSPL{%
-	\scoreitem{30}{[SSPL] Asking a person to step aside (\textit{smart} obstacle)}
-}%
-\ifNotSSPL{%
-	\scoreitem{30}{[DSPL \& OPL] Moving away movable object}
-}%
-
-	\setTotalScore{200}
+	\scoreheading{Main Goal}
+	\scoreitem{500}{Carry the bag to the car's location and coming back home}
+	\penaltyitem{-100}{help in finding the bag}
+	\penaltyitem{-100}{help in carrying the bag}
+	\penaltyitem{-200}{help in tracking the guest}
+	\penaltyitem{-100}{help in going back home}
+	
+	\scoreheading{Bonus rewards}
+	\scoreitem{100}{Find the bag by understanding the pointing behaviour}
+	\scoreitem{400}{Avoid all the obstacles outdoor}
```

Comment:
Split per obstacle

Reply:
Ok then it will better to have a fixed number of obstacles, say three, with 100pts for avoiding each.

Reply:
Depends on the nature of the obstacle. As responsible it is your job to analyze the technical difficulty of an straightforward solution and assign a reward proportional to other skills in the rulebook.

The number of obstacles is irrelevant as long as its position remains unknown to the robot.